src/hotspot/cpu/aarch64/aarch64_vector.ad

```diff
@@ -155,7 +155,7 @@ source %{
     }
 
     int length_in_bytes = vlen * type2aelembytes(bt);
-    if (UseSVE == 0 && length_in_bytes > 16) {
+    if (UseSVE == 0 && length_in_bytes > FloatRegister::neon_vl) {
```

Comment:
Should we also update `aarch64_vector_ad.m4` to avoid any mismatch :)

Reply:
Nice catch! I overlooked this place. Thanks for your reminder!